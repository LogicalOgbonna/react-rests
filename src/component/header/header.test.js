import React from "react";
import { shallow } from 'enzyme';
import Header from "./header";
import { findByTestAttr, renderWithoutCrashing } from "../../utils/tests";

const setUp = (props = {}) => {
    return shallow(<Header {...props} />);
}

describe('<Header /> Component', () => {
    let component;
    beforeEach(() => {
        component = setUp();
    })
    renderWithoutCrashing(component, "<Header />");
    it("Should Render a Logo", () => {
        const logo = findByTestAttr(component, "LogoImg");
        expect(logo.length).toBe(1);
    })
})
