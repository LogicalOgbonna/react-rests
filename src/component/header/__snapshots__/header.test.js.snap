// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Component should render <Header /> Without Crashing 1`] = `undefined`;
